app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss

```diff
@@ -7,6 +7,7 @@
   font-size: $font-size-small;
   font-weight: $font-weight-normal;
   position: relative;
+  box-shadow: 0px 2px 3px 0 rgba(0, 0, 0, 0.07);
```

Comment:
We have pre defined shadows in `mixins.scss`, Please reuse them

Reply:
What are your thoughts on this look of bubble shadows (I love it!) ...
`0 1px 3px $shadow-color-4; = 0 1px 3px rgba(0, 0, 0, .05);`

![Screen Shot 2020-07-01 at 7 04 11 PM](https://user-images.githubusercontent.com/7905931/86271727-b2c27680-bbcd-11ea-84ad-bb760fbaf1ae.png)

Reply:
I love this too :D But the contrast might be an issue for some screens and people,  So It might take a while to decide whether we should do this or not. So either we can move this to a separate PR so that the other changes are not affected.

Reply:
Alright ... I do so by just removing this line so that only the other changes are in effect?